DuckDuckGo/Configuration/ConfigurationManager.swift

```diff
@@ -163,3 +190,39 @@ struct ConfigurationManager {
     }
     
 }
+
+extension ConfigurationManager {
+    override var presentedItemURL: URL? {
+        return store.fileUrl(for: .privacyConfiguration).deletingLastPathComponent()
+    }
+
+    override func presentedSubitemDidAppear(at url: URL) {
+        guard url == store.fileUrl(for: .privacyConfiguration) else { return }
+        updateTrackerBlockingDependencies()
+    }
+
+    override func presentedSubitemDidChange(at url: URL) {
+        guard url == store.fileUrl(for: .privacyConfiguration) else { return }
+        updateTrackerBlockingDependencies()
+    }
+
+    func subscribeToLifecycleNotifications() {
+        NotificationCenter.default.addObserver(self, selector: #selector(addPresenter), name: UIApplication.didBecomeActiveNotification, object: nil)
```

Comment:
Per [file presenter documentation](https://developer.apple.com/documentation/foundation/nsfilecoordinator), we can use `willEnterForegroundNotification` here instead.